### PR TITLE
Update linux notifications function

### PR DESCRIPTION
**Pull request by sourque**

### CODE REVIEW

File: src/utility_linux.go [hunk 1]
```diff
@@ -24,7 +24,23 @@ func decodeString(fileContent string) (string, error) {
...
+			# If bus exists, Ubuntu >= 18
+			if [ -e /run/user/$uid/bus ]; then
+			    display="unix:path=/run/user/$uid/bus"
+			else
+			# Ubuntu <= 16
+			    display="unix:abstract=$(cat /run/user/$uid/dbus-session | cut -d '=' -f3)"
+			fi
+
+			sudo -u $user DISPLAY=:0 DBUS_SESSION_BUS_ADDRESS=$display notify-send -i /opt/aeacus/assets/logo.png "Aeacus SE" "` + messageString + `"`)
+	}
```
Comment (aaditgupta21) [inline src/utility_linux.go:43]: What if someone did put in the user in scoring.conf and they are on ubuntu 16

Comment (safinsingh) [reply src/utility_linux.go:43]: That's what it's checking for

Comment (sourque) [reply src/utility_linux.go:43]: `/run/user/$uid/bus` is a symlink that's only in the newer versions of Ubuntu, `-e` returns true if any file (of any type) exists at the specified path.

**Review by aaditgupta21 — APPROVED** [review]
Tested and Works

**Pull request merged**